example/lambdas/db-provision-user-database/tests/test-index.js

```diff
@@ -53,7 +53,13 @@ test.afterEach.always(async (t) => {
   await knex.raw(`drop user if exists "${t.context.expectedDbUser}"`);
 });
 
-test('provision user database creates the expected database', async (t) => {
+test.afterEach.always(async (t) => {
```

Comment:
There's an `afterEach.always` block already above this? I assume you can't have both

Reply:
That actually should be 477364b5a -- given this is a throwaway unit container, we don't want to kill the postgres state on a failed test: 

477364b5a

Reply:
Yes, and in fact you can have multiples of each per the docs:

https://github.com/avajs/ava/blob/master/docs/01-writing-tests.md

For peace of mind I just went and threw a few errors into the block and got the failures/counts I'd expect.   The databases are also cleaning up each time from what I've observed.